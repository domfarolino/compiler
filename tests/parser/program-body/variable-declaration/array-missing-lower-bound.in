program dominic is
  string array [: 40];
begin end program.
