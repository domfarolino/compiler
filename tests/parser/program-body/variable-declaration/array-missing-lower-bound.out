[1;31mLine 2: Expected number for array lower bound, but got: ':'
Line 2: Error parsing program body
[0m